app/src/main/java/com/bogdan/codeforceswatcher/features/users/UsersAdapter.kt

```diff
@@ -51,6 +52,11 @@ class UsersAdapter(
 
         with(items[position] as UserItem.User) {
             with(holder as UserViewHolder) {
+                if (avatar.substring(0, 6) != "https:") {
```

Comment:
This can be done much simpler by using `Picasso.get().load(LinkValidator.avatar(avatar)).into(ivAvatar)`.

Reply:
Nice! Updated.